app/assets/javascripts/spree/backend/account_confirm.js

```diff
@@ -0,0 +1,80 @@
+var AccountConfirm = function(htmlInfo) {
+  this.$fbAccounts = $(htmlInfo.fbAccounts);
+  this.$twitterAccounts = $(htmlInfo.twitterAccounts);
+  this.$fbContainer = $(htmlInfo.fbContainer);
+  this.$twitterContainer = $(htmlInfo.twitterContainer);
+  this.$modalButton = $(htmlInfo.modalButton);
+  this.$submitButton = $(htmlInfo.submitButton);
+  this.$errorMessage = $(htmlInfo.errorMessage);
+  this.$confirmMesage = $(htmlInfo.confirmMesage);
+};
+
+AccountConfirm.prototype.appendNames = function(container, names, msg) {
+  container.html(msg + names);
+};
+
+AccountConfirm.prototype.fetchNames = function(account) {
+  var names = [];
+  $.each(account, function(index, elem) {
+    $elem = $(elem);
+    if($elem.prop('checked')) {
+      names.push($elem.data('name'));
+    }
+  });
+  return names;
+};
+
+AccountConfirm.prototype.fetchAndAppendAccountNames = function(account, account_container, msg) {
+  var names = this.fetchNames(account);
+  if(names.length == 0) {
+    this.appendNames(account_container, names, '')
+  } else {
+    this.$submitButton.show();
+    this.$errorMessage.hide();
+    this.$confirmMesage.show();
+    $('[data-type=message]').html('Post will be posted on the following accounts:-');
+    this.appendNames(account_container, names, msg)
+  };
+};
+
+AccountConfirm.prototype.populateAccountsContainer = function() {
+  this.fetchAndAppendAccountNames(this.$fbAccounts, this.$fbContainer, 'Facebook - ');
+  this.fetchAndAppendAccountNames(this.$twitterAccounts, this.$twitterContainer, 'Twitter - ');
+};
+
+AccountConfirm.prototype.addErrorMessage = function() {
```

Comment:
Try using this method for toggle, this way you can remove L:32-L-35

Do not use jquery toggle, just check for visbility and then add conditions accordingly